MagneticField/src/LUXEMagneticField.cpp

```diff
@@ -15,13 +15,13 @@ auto exampleDipole = [](const std::array<double, 3> &v) {
 
 BField_t buildLUXEBField(const transformationPos_t& transformPos,
```

Comment:
Can we hide the BField functions into an interface inheritance structure? I don't really like the the `buildBField` function takes the BField as a global parameter (line 34)